.ci/build.sh

```diff
@@ -7,6 +7,7 @@ mkdir private _site public
 if [[ -n "$GCI_TOKEN" ]]; then
   python manage.py fetch_gci_task_data private
   python manage.py cleanse_gci_task_data private _site
+  rm -rf private/
```

Comment:
What's up with this?

Reply:
The private data collected on line 8 has been cleansed and pushed into `_site` , and is no longer needed.  It is what is causing the coala errors.  This is the first place to remove it.